hardware_layer/hardware_imu/src/yawDirect.cpp

```diff
@@ -35,8 +35,7 @@ int main(int argc, char **argv)
     read_sensors();
     msg.data = TO_DEG(-atan2(magnetom[1], magnetom[0]));
     chatter_pub.publish(msg);
-    ROS_INFO("%s %f", "send an imu message", TO_DEG(-atan2(magnetom[1], magnetom[0])));
-    // ROS_INFO("%d \n",temp);
+    ROS_INFO("%s %f", "yawDirect: send an imu message", TO_DEG(-atan2(magnetom[1], magnetom[0])));
```

Comment:
There was a bug in the code. To debug i removed ros::this_node from this line. But after finding the error i forgot to revert it. Sorry my bad.